packages/design-system/src/components/Form/Primitives/Input/Input.module.scss

```diff
@@ -0,0 +1,39 @@
+@use '~@talend/design-tokens/lib/tokens';
+@use '../sharedStyles' as Form;
+
+.input {
+  @include Form.baseInput();
+  height: Form.$standardInputHeight;
+}
+
+.button {
+  padding: 0 tokens.$coral-spacing-xs;
+  background: transparent;
+  color: tokens.$coral-color-neutral-icon-weak;
+  transition: all tokens.$coral-transition-instant;
+
+  &_readOnly {
```

Comment:
hoo

Reply:
I tried

Reply:
If I use regular BEM, `--` in this case, I have to use brackets in TSX for CSS modules to work (`styles["input--readonly"]` instead of the auto-completed `styles.input_readOnly`).